packages/search/src/features/reindex/reindex.ts

```diff
@@ -74,22 +76,26 @@ export const reindex = async () => {
         }
       }),
       onDrop(doc) {
-        throw new Error(
-          `Document ${doc.document.compositionId} couldn't be inserted`
-        )
+        droppedCompositionIds.push(doc.document.compositionId)
       }
     },
     {
       meta: true
     }
   )
+
   const t2 = performance.now()
   logger.info(
     `Finished reindexing to ${index} in ${((t2 - t1) / 1000).toFixed(
       2
     )} seconds`
   )
 
+  if (droppedCompositionIds.length) {
+    logger.warn(`Could not index ${droppedCompositionIds.length} document(s)`)
+    logger.warn(droppedCompositionIds.join(', '))
```

Comment:
```suggestion
    logger.error(`Could not index ${droppedCompositionIds.length} document(s)`)
    logger.error(droppedCompositionIds.join(', '))
```
This shouldn't happen, maybe wise to log it as error or even throw an error? What do you think

Reply:
This is fine for the 1.6 hotfix. Lets create a new issue where we incorporate your findings of the reindexing process

Reply:
https://github.com/opencrvs/opencrvs-core/issues/7743